subworkflows/local/bam_impute_quilt/tests/main.nf.test.snap

```diff
@@ -62,22 +62,22 @@
                         "id": "NA12878",
                         "panel": "1000GP",
                         "chr": "22",
-                        "chunk": "16570065-16597215",
+                        "chunk": "22:16570065-16597215",
```

Comment:
Why do chunks add the chromosome now?

Reply:
It's an old update to standardize with the other subworkflow that use in the meta map the region (chr:start-end) convention.

Reply:
The snapshot hasn't been updated when the change was made.